src/components/UserMessageSection.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 
-export function UserMessageDiv() {
+export function UserMessageSection() {
     return (
-        <div id="User-Message-Div">
+        <div id="user-message-section">
```

Comment:
do we need to attach an id to object like this? most of your interactions should be through props.

Reply:
Hey @pollend Thanks for reviewing :)

The user message div is just a box where we will add scrollable section where user's interaction will be done. It's a placeholder for now, that's why it's the only ID styled in the css stylesheet. It'll be replaced with more appropriate styling later. (I have started working on that) This PR is basically covering this part - https://trello.com/c/WvdibFBb/4-build-basic-layout-container (I should have mentioned it in the desc.  :)